autofit/non_linear/mcmc/zeus/zeus.py

```diff
@@ -248,6 +248,13 @@ def _fit(self, model: AbstractPriorModel, analysis, log_likelihood_cap=None):
 
         self.logger.info("Zeus sampling complete.")
 
+    def config_dict_with_test_mode_settings_from(self, config_dict):
+
+        config_dict["nwalkers"] = 50
+        config_dict["nsteps"] = 2
+
+        return config_dict

```

Comment:
You can also write:

```python
return {
    **config_dict,
    "nwalkers": 50,
    "nsteps": 2,
}
```

This is safer because it creates a new dictionary rather than mutating the existing one.